### DISCUSSION TITLE
What are the benefits of trpc-sveltekit over plain trpc?

### DISCUSSION THREAD

**Discussion by mateuszlewko** [icflorescu/trpc-sveltekit#104] [Q&A]
Hi there!

Many thanks for this project. I wonder what are the main benefits of using trpc-sveltekit over plain trpc with standalone server in SvelteKit? I see that we don't need to setup a separate http server. Could you summarise other benefits?

Thanks!
Mateusz

**Comment by icflorescu** [comment]
Less lines of code :-)
